src/openrpc/secure_storage.json

```diff
@@ -256,6 +256,168 @@
+      "tags": [
+        {
+          "name": "capabilities",
+          "x-manages": [
+            "xrn:firebolt:capability:storage:secure"
+          ]
+        }
+      ],
+      "params": [
+        {
+          "name": "appId",
+          "summary": "appId for which value is being set",
+          "schema": {
+            "type": "string"
+          },
+          "required": true
+        },
+        {
+          "name": "scope",
+          "summary": "The scope of the data key",
+          "schema": {
+            "$ref": "#/components/schemas/StorageScope"
+          },
+          "required": true
+        },
+        {
+          "name": "key",
+          "summary": "Key to set",
+          "schema": {
+            "type": "string"
+          },
+          "required": true
+        },
+        {
+          "name": "value",
+          "summary": "Value to set",
+          "schema": {
+            "type": "string"
+          },
+          "required": true
+        },
+        {
+          "name": "options",
+          "summary": "Optional parameters to set",
+          "schema": {
+            "$ref": "#/components/schemas/StorageOptions"
+          },
+          "required": false
+        }
+      ],
+      "result": {
+        "name": "success",
+        "schema": {
+          "const": null
+        }
+      },
+      "examples": [
+        {
+          "name": "Set a refresh token with name authRefreshToken with optional paramter for appId foo",
+          "params": [
+            {
+              "name": "appId",
+              "value": "foo"
+            },
+            {
+              "name": "scope",
+              "value": "device"
+            },
+            {
+              "name": "key",
+              "value": "authRefreshToken"
+            },
+            {
+              "name": "value",
+              "value": "VGhpcyBub3QgYSByZWFsIHRva2VuLgo="
+            },
+            {
+              "name": "options",
+              "value": { 
+                "ttl": 600
+              }
+            }
+          ],
+          "result": {
+            "name": "defaultResult",
+            "value": null
+          }
+        },
+        {
+          "name": "Set a refresh token with name authRefreshToken without optional parameter for appId foo",
+          "params": [
+            {
+              "name": "appId",
+              "value": "foo"
+            },
+            {
+              "name": "scope",
+              "value": "account"
+            },
+            {
+              "name": "key",
+              "value": "authRefreshToken"
+            },
+            {
+              "name": "value",
+              "value": "VGhpcyBub3QgYSByZWFsIHRva2VuLgo="
+            }
+          ],
+          "result": {
+            "name": "defaultResult",
+            "value": null
+          }
+        }
+      ]
+    },
+    {
+      "name": "removeAll",
```

Comment:
we should call this `removeAllForApp` and have a second method called `removeAll` that doesn't include the `appId` parameter and `uses` the capability.

I also think `clear` might be a better name, but that's subjective.